### PR TITLE
Added Tuya plugin

### PR DESCRIPTION
**Pull request by galadril**
Added Tuya plugin

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -527,10 +527,18 @@
...
+        "description": "Controls TUYA devices your network (mainly on/off switches and Lights).",
+        "folder": "tuyaha",
+        "name": "Tuya",
+        "repository": "https://github.com/guino/tuyaha"
+    },
     "WuDirect": {
         "author": "vaneeten",
         "branch": "master",
-        "description": "Direct reception and parsing of Wunderground-formatted Weatherstation data.",
+        "description": "Controls TUYA devices your network (mainly on/off switches and Lights).",
```
Comment (stas-demydiuk) [inline plugins.json:541]: @galadril Looks like by accident you've updated the other plugin as well

Comment (galadril) [reply plugins.json]: Woops good spot.. should be fixed now

**Comment by galadril** [comment]
Should be good now, thanks for the feedback

**Comment by stas-demydiuk** [comment]
Thanks!

**Pull request merged**